### ISSUE TITLE
Wings stop working after teleport/dimension change

### ISSUE THREAD

**Issue by MCaesar** [adriantodt/Winged#41]
On multiple occasions the wings have stopped working, usually after a dimension change. Regardless of how I go to the End, I would have to relog to get them working again. However they also will stop working after a teleport to another player in the same dimension. It occurs randomly, not due to a specific command/item/dimension (End is usually every time though). In most cases, relogging would fix the issue, however if it doesn't work, the only way to get them working again is to die.
No crash logs or information in logs with reference to the Winged mod.

**Comment by Mendanbar** [comment]
I'm having the same issue.  Using the All of Fabric 3 modpack.  I've also noticed that the enable/disable elytra key stops working as well when the wings stop.  Seems to imply that that the wings are acting as though they aren't there.  The model is still visibly present on my character though.

**Comment by Anti27** [comment]
+1
TP does work on same dimension, but if made to overworld creative fly is gone

**Comment by NotJustAnna** [comment]
This is a FallFlyingLib bug, which (hopefully) be superseded by an official Fabric API module: https://github.com/FabricMC/fabric/pull/1815

Once the module is merged, I'll work on a version that uses the Fabric API module and this will (hopefully²) be fixed.

**Comment by Gamemassa** [comment]
Just an FYI that this issue is still happening occasionally.
